### ISSUE TITLE
LearningRateScheduleCallback/LearningRateWarmupCallback missing from TF api

### ISSUE THREAD

**Issue by Troy-M** [uber/horovod#322]
The keras api has the functionality to use the "warm up" learning rate approach straight out of the box, but the tensorflow verison of horovod does not. Could this be achieved using the tf.train.SessionRunHook system?

**Comment by alsrgv** [comment]
@Troy-M, you can define learning rate as a function of global step with a few `tf.cond()` statements.  Something like `https://github.com/tensorflow/benchmarks/blob/master/scripts/tf_cnn_benchmarks/benchmark_cnn.py#L978`.
